tehuti.py

```diff
@@ -168,6 +170,45 @@ def run(self):
         return [sum(vals) / self.number for vals in self.usage_log]
 
 
+class AccuracyMetric(Metric):
+    class Context(object):
+        pass
+
+    def __init__(self, body, setup=None, repeat=100, number=1, name=None):
```

Comment:
I can't think of any (sane) use case that would need this. Better to keep it simple. Less code = less bugs. :wink: